contrib/packaging/macos.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# make sure starting from project's root
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd "$SCRIPT_DIR/../.."
+
+python3 -m pip install pyinstaller
+
+make clean
+rm -rf build/portable/
+mkdir -p build/portable/
+cp -R ./modmesh build/portable/
+cp -R ./resources build/portable/
+cd build/portable/
+
+cat > modmesh.py <<'EOF'
+import os
+os.environ["QT3D_RENDERER"] = "opengl"
+from modmesh.pilot import launch
+if __name__ == "__main__":
+    launch()
+EOF
+
+pyinstaller --onedir \
```

Comment:
Would `pilot` work with this installation?  `pilot` is the primary entry point of modmesh.